src/components/AddForm.tsx

```diff
@@ -0,0 +1,75 @@
+import { Box, Button, TextField } from "@mui/material";
+import { Textarea } from "@mui/joy";
+import { useForm } from "react-hook-form";
+import { z } from "zod";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { v1 } from "uuid";
+import { TodosStore } from "../store/todosStore";
+interface AddFormProps {
+  handleClose: () => void;
+}
+
+interface AddFormValues {
+  taskName: string;
+  taskDescription?: string;
+}
```

Comment:
```suggestion
interface AddFormValues {
  name: string;
  description?: string;
}
```
Мне кажется тут излишне конкретизация. Это всего лишь 2 поля и из контекста должно быть понятно, что это название таски.